src/scripts/components/header.ts

```diff
@@ -0,0 +1,29 @@
+export class Header {
+	private readonly closeBTN: Element | null;
```

Comment:
change this from `closeBTN` to `closeButton`

Reply:
Done